pennylane/transforms/mitigate.py

```diff
@@ -45,6 +45,12 @@ def fold_global(tape: QuantumScript, scale_factor) -> tuple[QuantumScriptBatch,
 
     .. seealso:: :func:`~.pennylane.transforms.mitigate_with_zne`; This function is analogous to the implementation in ``mitiq``  `mitiq.zne.scaling.fold_global <https://mitiq.readthedocs.io/en/v.0.1a2/apidoc.html?highlight=global_folding#mitiq.zne.scaling.fold_global>`_.
 
+    .. note::
+
+        This method no longer decomposes the circuit as part of the folding procedure. Users are encouraged to use
+        :func:`~.pennylane.transforms.compile` to decompose the circuit into a target gateset before using this.
```

Comment:
```suggestion
        :func:`~.pennylane.transforms.decompose` to decompose the circuit into a target gateset before using this.
```

Given compile does a lot of other things as well as decompose, it might be better to recommend the strictly decomposition transform.

Reply:
I couldn't find `pennylane.transforms.decompose` which is why I used `pennylane.transforms. compile` 😅

Reply:
@obliviateandsurrender I think this feature should be on `master`!

Reply:
Getting added in the next day or two. #6334

Reply:
@obliviateandsurrender, #6334 has been merged!